lib/vmdb/loggers.rb

```diff
@@ -23,73 +23,38 @@ def self.apply_config(config)
-      apply_config_value(config, $redfish_log,        :level_redfish)
-      apply_config_value(config, $rhevm_log,          :level_rhevm)
-      apply_config_value(config, $scvmm_log,          :level_scvmm)
-      apply_config_value(config, $vcloud_log,         :level_vcloud)
-      apply_config_value(config, $vim_log,            :level_vim)
       apply_config_value(config, $remote_console_log, :level_remote_console)
-    end
 
-    private_class_method def self.create_loggers
-      path_dir = ManageIQ.root.join("log")
-
-      $container_log      = create_container_logger
-      $journald_log       = create_journald_logger
-      $log                = create_multicast_logger(path_dir.join("evm.log"))
-      $rails_log          = create_multicast_logger(path_dir.join("#{Rails.env}.log"))
-      $audit_log          = create_multicast_logger(path_dir.join("audit.log"), AuditLogger)
-      $api_log            = create_multicast_logger(path_dir.join("api.log"))
-      $ansible_tower_log  = create_multicast_logger(path_dir.join("ansible_tower.log"))
-      $miq_ae_logger      = create_multicast_logger(path_dir.join("automation.log"))
-      $aws_log            = create_multicast_logger(path_dir.join("aws.log"))
-      $azure_log          = create_multicast_logger(path_dir.join("azure.log"), ProviderSdkLogger)
-      $azure_stack_log    = create_multicast_logger(path_dir.join("azure_stack.log"))
-      $cn_monitoring_log  = create_multicast_logger(path_dir.join("container_monitoring.log"))
-      $datawarehouse_log  = create_multicast_logger(path_dir.join("datawarehouse.log"))
-      $fog_log            = create_multicast_logger(path_dir.join("fog.log"), FogLogger)
-      $gce_log            = create_multicast_logger(path_dir.join("gce.log"))
-      $ibm_cloud_log      = create_multicast_logger(path_dir.join("ibm_cloud.log"), ProviderSdkLogger)
-      $ibm_terraform_log  = create_multicast_logger(path_dir.join("ibm_terraform.log"), ProviderSdkLogger)
-      $kube_log           = create_multicast_logger(path_dir.join("kubernetes.log"))
-      $lenovo_log         = create_multicast_logger(path_dir.join("lenovo.log"))
-      $nsxt_log           = create_multicast_logger(path_dir.join("nsxt.log"))
-      $nuage_log          = create_multicast_logger(path_dir.join("nuage.log"))
-      $policy_log         = create_multicast_logger(path_dir.join("policy.log"))
-      $redfish_log        = create_multicast_logger(path_dir.join("redfish.log"))
-      $rhevm_log          = create_multicast_logger(path_dir.join("rhevm.log"))
-      $scvmm_log          = create_multicast_logger(path_dir.join("scvmm.log"))
-      $vcloud_log         = create_multicast_logger(path_dir.join("vcloud.log"))
-      $vim_log            = create_multicast_logger(path_dir.join("vim.log"))
-      $remote_console_log = create_multicast_logger(path_dir.join("remote_console.log"))
+      # TODO: Move this into the manageiq-api plugin
```

Comment:
This is because of https://github.com/ManageIQ/manageiq-api/blob/5683d9ebd1d708d0be3257296d081595c203f1ac/lib/manageiq/api/engine.rb#L9-L18, which I can solve eventually, but am skipping for now.

Reply:
I assume that this comment is here prior to the suggestion by @jrafanie to only do a subset of loggers first:

https://github.com/ManageIQ/manageiq/pull/20950#issuecomment-759784345

And this is less relevant of a "FYI" now that  we are really only moving out two of the loggers for this first pass, correct?